transformers/events/DOCUMENTATION.md

```diff
@@ -2,116 +2,206 @@
+       * The second parameter passed to `tend` will be `topic[3]`.
+       * Any additional parameters will be in the log's data field.
+       * More detail is located in the [DSNote
+         repo](https://github.com/dapphub/ds-note).
+       * For an example implementation look at the `flip_sol` transformer.
+   1. If, while implementing the transformer and its corresponding unit tests
+      you find you need to migrate the database see below. 
+
+### Store the data in the database
+1. Use the `make new_migration` task to create a new migration.
+    * Each event log has its own table in the database.
+    * The specific log event tables are all created in the `maker` schema.
+1. The new migration can be run by running `make test` or `make migration NAME=vulcanize_test`.
+1. To verify that the migration will work create an integration for the shiny
+   new transformer in [`integration_tests`](../integration_tests).
+1. Integration tests can be run with `make integrationtest`.
+
+### Add the transformer to the list of transformers to run
+Finally you can add the transformer to the list of transformers by updating the
+configuration, and creating an initializer for the transformer.
+
+1. In the environments (docker.toml, testing.toml and mcdTransformers.toml) add
+   the new package name the list of `transformerNames` in the exporter - alphabetically.
+1. Underneath that list add a configuration (alphabetically again) to list of
+   configurations. For example:
+   
+``` toml
+[exporter.log_delete]
+      path = "transformers/events/log_delete/initializer"
+      type = "eth_event"
+      repository = "github.com/makerdao/vdb-mcd-transformers"
+      migrations = "db/migrations"
+      contracts = ["OASIS_MATCHING_MARKET_ONE", "OASIS_MATCHING_MARKET_TWO"]
+      rank = "0"
+```
+1. Note the path to the initializer is a directory named initializer in your new
+   event transformer package. Of course you haven't created that yet. Create
+   that directory and inside it create a file named `initializer.go`. 
+1. Initializers are boilerplate that tell the system how to create your
+   transformer. They look like this:
+   
+```go
+package initializer
+
+import (
+  "github.com/makerdao/vdb-mcd-transformers/transformers/events/log_delete"
+  "github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+  "github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+  "github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+)
+
+var EventTransformerInitializer event.TransformerInitializer = event.ConfiguredTransformer{
+  Config:      shared.GetEventTransformerConfig(constants.LogDeleteTable, constants.LogDeleteSignature()),
+  Transformer: log_delete.Transformer{},
+}.NewTransformer
+```
+
+Simply replace the constants and package names with your transformer.
+
+1. Finally add your package to the list of transformerExporters in `plugins/transformerExporter.go`. Again alphabetically.
```

Comment:
maybe extraneous but you can generate the `transformerExporter.go` from the command line with `./vulcanizedb compose --config=/path/to/config.toml`

Reply:
Didn't know that!